charts/factorio/values.yaml

```diff
@@ -68,6 +68,13 @@ service:
   # externalTrafficPolicy: Cluster
   annotations: {}
 
+#### Deployment Configuration ####
+## @section Deployment Parameters
+## @param deployment.hostNetwork Enable hostNetwork for the deployment in order to use hostPort
+## @param deployment.hostPort Specify the hostPort you want to use for the game (default port 34179 is recommended)
+deployment:
+  hostNetwork: false
+  hostPort: 31479
```

Comment:
If this _is_ necessary, let's default it to what factorio normally uses which is 34197. The NodePort uses 31479 because that's what's in range of the node port block kube defines.

Reply:
Yes, totally. Oh man I was copy pasting those numbers a lot, so I got confused.